client/ayon_core/hosts/houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -0,0 +1,95 @@
+import os
+import pyblish.api
+
+
+class CollectLocalRenderInstances(pyblish.api.InstancePlugin):
+    """Collect instances for local render.
+
+    Agnostic Local Render Collector.
+    """
+
+    # this plugin runs after Collect Render Products
+    order = pyblish.api.CollectorOrder + 0.12
+    families = ["mantra_rop",
+                "karma_rop",
+                "redshift_rop",
+                "arnold_rop",
+                "vray_rop"]
+
+    hosts = ["houdini"]
+    label = "Collect local render instances"
+
+    def process(self, instance):
+
+        if instance.data["farm"]:
+            self.log.debug("Render on farm is enabled. "
+                           "Skipping local render collecting.")
+            return
+
+        # Create Instance for each AOV.
+        context = instance.context
+        expectedFiles = next(iter(instance.data["expectedFiles"]), {})
+
+        product_type = "render"  # is always render
+        product_group = "render{Task}{productName}".format(
+            Task=self._capitalize(instance.data["task"]),
+            productName=self._capitalize(instance.data["productName"])
+        )  # is always the group
+
+        for aov_name, aov_filepaths in expectedFiles.items():
+            # Some AOV instance data
+            # label = "{productName}_{AOV}".format(
+            #     AOV=aov_name,
+            #     productName=instance.data["productName"]
+            # )
+            name_template = "render{Task}{productName}_{AOV}"
+            if not aov_name:
+                # This is done to remove the trailing `_`
+                # if aov name is an empty string.
+                name_template = "render{Task}{productName}"
+
+            product_name = name_template.format(
+                Task=self._capitalize(instance.data["task"]),
+                productName=self._capitalize(instance.data["productName"]),
+                AOV=aov_name
+            )
+
+            # Create instance for each AOV
+            aov_instance = context.create_instance(product_name)
+
+            # Prepare Representation for each AOV
+            aov_filenames = [os.path.basename(path) for path in aov_filepaths]
+            staging_dir = os.path.dirname(aov_filepaths[0])
+            ext = aov_filepaths[0].split(".")[-1]
+
+            aov_instance.data.update({
+                # 'label': label,
+                "task": instance.data["task"],
+                "folderPath": instance.data["folderPath"],
+                "frameStart": instance.data["frameStartHandle"],
+                "frameEnd": instance.data["frameEndHandle"],
+                "productType": product_type,
+                "productName": product_name,
+                "productGroup": product_group,
+                "tags": [],
+                "families": ["render.local.hou"],
```

Comment:
Maybe you can try to add review as being parts of families to see if it works as expected.
```suggestion
                "families": ["render.local.hou", "review"],
```

Reply:
> So, After a quick investigation. Review product logic in Houdini was built assuming we are only publishing reviews from review instances. (opengl node)

I recommend you test on the local publishing in Nuke Host or study the logic behind the submit publish job. They are helpful for you to work on this PR.

Reply:
I'm going to dig more. I was just mentioning that the logic is interlaced. 
All review plugins in Houdini are not generic as they expect opengl node. 
and in order to make it work. I needed to add to them something like this
```python 
ropnode = hou.node(instance.data.get("instance_node"))
        if ropnode.type().name() != "opengl":
            self.log.debug("Skipping OpenGl extraction. Rop node {} "
                           "is not of OpenGl type.".format(ropnode.path()))
            return
```

Reply:
@moonyuet  I'm not sure about my approach to implement it. so, I created this PR.
https://github.com/ynput/ayon-core/pull/381

Reply:
> @moonyuet I'm not sure about my approach to implement it. so, I created this PR. #381

it works nicely with local rendering and I already merged it but I spotted another issue with the sync of the settings from creator attr to instance attr. I will inform with more detail later.